jsonschema/validators.py

```diff
@@ -76,12 +77,13 @@ def __init__(
             self._types.update(types)
 
             if resolver is None:
+                _schema = schema
                 if schema is True:
-                    resolver = RefResolver.from_schema({})
+                    _schema = {}
```

Comment:
The problem here is that now you will mis-report schemas.

One major gap in draft 6 support is writing some tests for the new schema coersion behavior (of bools) -- specifically, this will misreport what the schema was in `ValidationError.schema`.